src/SpotifyApi.NetCore/PlayerApi.cs

```diff
@@ -778,6 +778,19 @@ public async Task TransferPlayback(string deviceId, bool? play = null, string ac
                 accessToken: accessToken);
         }
 
+
+        #endregion
+
+
+        #region AddToQueue
+        public async Task AddToQueue(string itemUri, string accessToken = null, string deviceId = null)
+        {
+            var builder = new UriBuilder($"{BaseUrl}/me/player/queue");
+            builder.AppendToQueryIfValueNotNullOrWhiteSpace("device_id", deviceId);
+            builder.AppendToQueryIfValueNotNullOrWhiteSpace("uri", itemUri);
```

Comment:
Please add validation for `itemUri`, e.g.

    if (string.IsNullOrEmpty(itemUri)) throw new ArgumentNullException(nameof(itemUri));

Please put `accessToken` parameter last.

Reply:
Okay, I have added the variable check and added some test cases